www/tenfourfox/Portfile

```diff
@@ -4,10 +4,12 @@ PortSystem              1.0
 PortGroup               github 1.0
 
 categories              www
-maintainers             {@barracuda156 gmail.com:vital.had} openmaintainer
+maintainers             {@barracuda156 gmail.com:vital.had} {@BlackBirdLC netcourrier.com:thewireless} openmaintainer
 license                 GPL-2
 name                    tenfourfox
 subport                 tenfourfox-devel {}
+subport                 aquafox {}
+
```

Comment:
It is better not to add unneeded duplicate empty lines.

Reply:
An oversight on my part.